localization/de/strings.po

```diff
@@ -51,6 +51,15 @@ msgid_plural "%s Products"
 msgstr[0] "%s Produkt"
 msgstr[1] "%s Produkte"
 
+msgid "Standard Consume-Amount for this Product"
```

Comment:
Localizations are managed via Transifex as [mentioned in README](https://github.com/grocy/grocy#localization), the only (source) file which need to be updated with new strings is `/localization/strings.pot`